pkg/proxy/apis/config/types.go

```diff
@@ -188,6 +190,14 @@ const (
 	ProxyModeKernelspace ProxyMode = "kernelspace"
 )
 
+// LocalMode represents modes to detect local traffic from the node
+type LocalMode string
+
+// Currently supported modes for LocalMode
+const (
+	LocalModeClusterCIDR LocalMode = "clustercidr"
```

Comment:
Usually our string constants match a substring of the Go identifier, unless there's a spec that dictates otherwise (e.g. when it has to be a FQDN or something like that). For example see how the `PersistentVolumeReclaimPolicy` constants are capitalized and match the last word of the Go identifier:

https://github.com/kubernetes/kubernetes/blob/e3ab253cd8193d348b785aed825072b5e50a3b8e/staging/src/k8s.io/api/core/v1/types.go#L349-L372

Reply:
Configs historically haven't been consistent, but I think we should try to be consistent with the style of the core APIs whenever possible.

Reply:
Ack. So I'm assuming "ClusterCIDR" would work? Will change. Note this will also be the value the users will enter in the kube configuration YAML for kubeadm.

Reply:
yes